### DISCUSSION TITLE
Branch Name on Pull Request URL

### DISCUSSION THREAD

**Discussion by yamil777** [community/community#141587] [Pull Requests] [answered]
In the URL of the pull request page, it would be beneficial to include the branch name. Currently, there is a copy button to capture the branch name, but having it readily available in the URL would simplify the process for developers. This change would allow for quick copying and switching between branches, eliminating the need for multiple clicks.

**Comment by queenofcorgis** [comment, marked as answer]
Hey there!  👋 

Thanks for posting in the GitHub Community, @yamil777 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
